src/components/ec-blog-item/ec-blog-item.spec.js

```diff
@@ -0,0 +1,18 @@
+import { mount } from '@vue/test-utils';
+import EcBlogItem from './ec-blog-item.vue';
+
+describe('EcBlogItem', () => {
+  function mountEcBlogItem(props, mountOpts) {
+    return mount(EcBlogItem, {
+      propsData: {
+        ...props,
+      },
+      ...mountOpts,
+    });
+  }
+
+  it('should render post', () => {
+    const wrapper = mountEcBlogItem();
+    expect(wrapper.element).toMatchSnapshot();
```

Comment:
We should pass the props to check in the snapshots if we are rendering those texts